openpype/tools/publisher/widgets/overview_widget.py

```diff
@@ -1,4 +1,4 @@
-from qtpy import QtWidgets, QtCore
+from qtpy import QtWidgets, QtCore, QtGui
```

Comment:
Why you use QtGui?

Reply:
OK, well I will try to use the orignial version so maybe we used the previous version as well